macros/audit.sql

```diff
@@ -55,12 +62,22 @@
 
 
 {% macro create_audit_schema() %}
-    create schema if not exists {{ logging.get_audit_schema() }}
+    {% do create_schema(api.Relation.create(
+        database=target.project,
```

Comment:
```suggestion
        database=target.database,
```
This is what's causing the integration test error currently. `target.project` aliases to `target.database` for the `dbt-bigquery` plugin only.